private/buf/bufformat/testdata/proto2/option/v1/option_name.golden.proto

```diff
@@ -11,12 +11,12 @@ extend google.protobuf.FieldOptions {
 }
 
 message Foo {
-  option (/* One */something/* Two */) = 1;
+  option (/* One */ something /* Two */) = 1;
 
   string name = 1 [
     // Leading comment on deprecated.
-    deprecated/* After deprecated */ = true,
+    deprecated /* After deprecated */ = true,
     // Leading comment on another.
-    (/* One */another/* Two */) = 2
+    ( /* One */ another /* Two */ ) = 2
```

Comment:
nit: Looks nicer here if there aren't added spaces added after '(' and before ')'.

```suggestion
    (/* One */ another /* Two */) = 2
```